setup.py

```diff
@@ -1,18 +1,43 @@
 #!/usr/bin/python3
 import setuptools
+import sys
 import PyInstaller.__main__ as pim
 
-pim.run([
-	"--name=DodgeToDrums",
-	"--onefile",
-	"--windowed",
-	"--clean",
-	"--exclude-module=tkinter",
-	"--exclude-module=PyQt4",
-	"--exclude-module=PyQt5",
-	"--exclude-module=numpy",
-	"--hidden-import=packaging.requirements",
-#	"--icon=icon.ico",
-#	"--add-data=./data/",
-	"main.py"
-	])
+DEFAULT_PROFILE = "dist"
+
+if len(sys.argv) > 0:
+	PROFILE = sys.argv[1]
+else:
+	PROFILE = DEFAULT_PROFILE
+
+PROFILES = {
+	"dist": [
+		"--name=DodgeToDrums",
+		"--onefile",
+		"--windowed",
+		"--clean",
+		"--exclude-module=tkinter",
+		"--exclude-module=PyQt4",
+		"--exclude-module=PyQt5",
+		"--exclude-module=numpy",
+		"--hidden-import=packaging.requirements",
+	#	"--icon=icon.ico",
+	#	"--add-data=./data/",
+		"main.py"
+	],
+	"dev": [
+		"--name=DodgeToDrums",
+		"--onefile",
+		"--clean",
+		"--exclude-module=tkinter",
+		"--exclude-module=PyQt4",
+		"--exclude-module=PyQt5",
+		"--exclude-module=numpy",
+		"--hidden-import=packaging.requirements",
+	#	"--icon=icon.ico",
+	#	"--add-data=./data/",
+		"main.py"
+	]
+}
```

Comment:
I think just adding and subtracting to/from a standard argument set would be best here.
something like 
```python3
pargs=[…]
if mode=="dev":
  pargs.append("--console")
else:
  pargs.append("--windowed")
```

Reply:
I thought about that, but I just wanted to get a simple prototype running and also for this a list xor would be necassary (to toggle the flags)